_source/_includes/footer.html

```diff
@@ -84,3 +84,5 @@ <h4 class="column-header">More Info</h4>
     {% if layout.js %}
       {% js '{{layout.js}}' %}
     {% endif %}
+    <!-- Fix <details> for IE/Edge -->
+    <script src="//cdn.jsdelivr.net/npm/details-polyfill@1/index.min.js" async></script>
```

Comment:
Is this the right place to put new scripts @rdegges @hamzaessaoui-okta?

Reply:
No response means yes! :)

Reply:
Sorry @mraible , just seeing this :)

but yes, that's a good place to load js